### ISSUE TITLE
Lifespan Events don't work with propagation_auth_headers=True argument

### ISSUE THREAD

**Issue by Fedir-Yatsenko** [epam/ai-dial-sdk#56]
If I set the _propagation_auth_headers_ argument in the DIALApp constructor and add one of the FastAPI _lifespan events_, the application will fail.

Code:
```Python
app = DIALApp('...', propagation_auth_headers=True)  # lifespan=lifespan)
app.add_chat_completion("echo", EchoApplication())

@app.on_event("startup")
async def startup_event():
    print('!!! Startup event !!!')

# Run builded app
if __name__ == "__main__":
    uvicorn.run(app, port=5000, host="0.0.0.0", lifespan='on')
```

Error message:
> INFO:     Started server process [4112]
> INFO:     Waiting for application startup.
> ERROR:    Exception in 'lifespan' protocol
> Traceback (most recent call last):
>  File "...\venv\lib\site-packages\uvicorn\lifespan\on.py", line 86, in main
>    await app(scope, self.receive, self.send)
>  File "...\venv\lib\site-packages\uvicorn\middleware\proxy_headers.py", line 84, in __call__
>    return await self.app(scope, receive, send)
>  File "...\venv\lib\site-packages\fastapi\applications.py", line 1106, in __call__
>    await super().__call__(scope, receive, send)
>  File "...\venv\lib\site-packages\starlette\applications.py", line 122, in __call__
>    await self.middleware_stack(scope, receive, send)
>  File "...\venv\lib\site-packages\starlette\middleware\errors.py", line 149, in __call__
>    await self.app(scope, receive, send)
>  File "...\venv\lib\site-packages\aidial_sdk\header_propagator.py", line 26, in __call__
>    for header in scope["headers"]:
> KeyError: 'headers'
> ERROR:    Application startup failed. Exiting.
> 
> Process finished with exit code 3

**Comment by vladisavvv** [comment]
Hi @FedorYatsenko! Thank you for the report, I will check this.

**Comment by vladisavvv** [comment]
@FedorYatsenko Hi, it will be fixed in the next `0.7.0` release.

**Comment by Fedir-Yatsenko** [comment]
Thank you so much!
